### ISSUE TITLE
No instructions on compiling & running tests

### ISSUE THREAD

**Issue by TibbsTerry** [npgsql/doc#220]
Are there any details regarding contributing? and or getting the solution to compile etc.?

I've encountered various issue trying to compile & run unit tests
(Win10, VS2022, branch stable)

- Solution file format VS2019 won't work because of no support for .Net6 SDK
- Nuget configuration missing/wrong 
Error	NU1507	There are 5 package sources defined in your configuration. When using central package management, please map your package sources with package source mapping (https://aka.ms/nuget-package-source-mapping) or specify a single package source.
Adding Nuget.config from main branch seems to fix this
- Tests rely on Postgres installed locally
No instructions on on this, eg Postgres versions needed
No details on 'npgsql_tests' role needed eg permissions. (or a script to create the role)
- 6117 of 16826 tests fail
some examples:

Npgsql.EntityFrameworkCore.PostgreSQL.CommandInterceptionNpgsqlTestBase+CommandInterceptionNpgsqlTest.Intercept_non_query_one_app_and_one_injected_interceptor(async: False)
Npgsql.PostgresException : 53300: sorry, too many clients already

Npgsql.EntityFrameworkCore.PostgreSQL.Scaffolding.NpgsqlDatabaseModelFactoryTest.Column_collation_is_set
Npgsql.PostgresException : 42809: "BlogsView" is not a table

Npgsql.EntityFrameworkCore.PostgreSQL.Storage.LegacyNpgsqlTypeMappingTest.GenerateSqlLiteral_returns_timestamptz_datetime_literal
Assert.StartsWith() Failure:
Expected: TIMESTAMPTZ '1997-12-17 07:37:16+10:00
Actual:   TIMESTAMPTZ '1997-12-17 07:37:16+11:00...

**Comment by roji** [comment]
Yeah, we definitely should add some documentation. [There's this page](https://www.npgsql.org/doc/dev/tests.html) for setting up PG for the tests, but it's insufficient.

Here are some immediate answers, I'll try to add real docs at some point on all of this.

* I haven't updated the "stable" branch so much in recent times. To work on a patch release, check out the latest hotfix/6.0.x branch and work on that. It shouldn't have the nuget.config issue you mentioned above.
* For the "sorry, too many clients already", you'll need to bump up your PostgreSQL max_connection setting; 200 should do it.
* For the last two errors, these shouldn't happen - can you plaese open two individual issues in the EFCore.PG repo to track these?
